README.md

```diff
@@ -209,6 +209,10 @@ $ VO_BIT=1 make CONF=linux-x86_64-normal-server-$DEBUG_LEVEL THIRD_PARTY_HEAP=$P
 
 ## Test
 
+We support most MMTk GC plans in OpenJDK, such as `SemiSpace`, `Immix`, `GenImmix`, `StickyImmix`, `GenCopy`, `MarkCompact`, `MarkSweep`, `PageProtect`, and `NoGC`.
+When you run with the OpenJDK build from previous steps, you can choose one of the plans by setting an environment variable `MMTK_PLAN`. In the following examples, we use
+the generational immix plan, `GenImmix`.
```

Comment:
Maybe we should mention how to find out the plan available by linking to mmtk-core code?

Reply:
Maybe we can expose some information via, for example, `java --version` or `-XX:ShowMmtkPlans`, by printing out all plans supported by the OpenJDK binding, and/or all available plans in mmtk-core.

Reply:
I added a link to https://docs.mmtk.io/api/mmtk/util/options/enum.PlanSelector.html.